### ISSUE TITLE
Per-inverter detection of Calibration mode

### ISSUE THREAD

**Issue by gcoan** [springfall2008/batpred#1009]
**Is your feature request related to a problem? Please describe.**
I had an inverter replaced today so the engineer needed to recalibrate just the batteries connected to that inverter.  The other inverter with connected batteries was still left running (and in fact had been controlled quite happily by predbat whilst I only had a single working inverter/battery setup).

So one inverter was discharging the battery, the second inverter was in Idle and exporting excess solar, waiting for house load.

When I reconfigured predbat to add the first inverter back in, it detected that inverter 0 was in calibration mode and inverter 1 wasn't, but predbat put itself into Calibration mode for all inverters and stopped working even though it could still have controlled inverter 1 successfully.

**Describe the solution you'd like**
1. For multi-inverter setups, clearer information in predbat status as to which inverter is in calibration mode
2. If some inverters are not in calibration mode, continue controlling the non-calibrating inverters

**Describe alternatives you've considered**
Control the calibration outside of predbat, editing apps.yaml to remove the calibrating inverter. Put it back into predbat once calibration is complete.  Whilst for my case of an engineer-initiated calibration this would be OK,  for other situations where the battery/inverter randomly decides to go into calibration mode (which Predbat is now highlighting much more clearly than before), it would be useful for multi-inverter owners to be able to still use one battery/inverter.

Recognise this is a low priority edge case as most people don't have multiple inverters and battery calibrations are relatively rare.

**Additional context**
Logfile showing what happened
[calibrationlog.txt](https://github.com/springfall2008/batpred/files/15120213/calibrationlog.txt)
